docs/source/restore/index.rst

```diff
@@ -6,162 +6,79 @@ Restore
-In fact you can use any Scylla node with Scylla Manager Agent installed that has access to the backup location for getting the parameters.
-Note that there is no need for Scylla Manager Server installation prior to restore.
+Restore types
+=============
 
-This procedure can only restore complete cluster data, not a subset of keyspace or tables.
+Restore task has to be one of two types:
 
-Ansible
-=======
-
-Cloning the cluster is automated with Ansible playbook.
-The playbook has a `readme <https://github.com/scylladb/scylla-manager/tree/master/ansible/restore>`_ that explains how to use it.
-For reader's convenience it's repeated here.
-
-Getting the playbook
---------------------
-
-**Procedure**
-
-#. Clone repository from GitHub ``git clone git@github.com:scylladb/scylla-manager.git``
-#. Go to playbook directory ``cd scylla-manager/ansible/restore``
-
-Playbook parameters
--------------------
-
-All restore parameters shall be put to ``vars.yaml`` file.
-Copy ``vars.yaml.example`` as ``vars.yaml`` and change parameters to match your clusters.
-
-.. note:: make sure to update your ``var.yaml`` so the IPs of the **destination cluster** map to UUIDs of nodes of the **source cluster**.
-
-
-For example, for a source cluster
-
-.. code::
-
-   > nodetool status
-   
-   Datacenter: datacenter1
-   =======================
-   Status=Up/Down
-   |/ State=Normal/Leaving/Joining/Moving
-   --  Address      Load       Tokens       Owns    Host ID                               Rack
-   UN  10.240.0.6   119.6 MB   256          ?       9b6b5206-63c8-4cdf-9b4f-23e37977fd14  rack1
-   UN  10.240.0.93  122.69 MB  256          ?       2d8045e7-51a4-42c4-bb8f-5e8aa77a8228  rack1
-
-and a Destination cluster:
-
-.. code::
-
-   > nodetool status
-   
-   Datacenter: us-east1
-   ====================
-   Status=Up/Down
-   |/ State=Normal/Leaving/Joining/Moving
-   --  Address        Load       Tokens       Owns    Host ID                               Rack
-   UN  75.196.92.133  122.14 MB  256          ?       a21b38b4-c0b1-49f0-89f2-931fe027d6b0  b
-   UN  75.237.179.49  121.22 MB  256          ?       a0a63682-95e3-4cf3-9f46-4d551bd7eba2  b
+  * :doc:`restore tables <restore-tables>` - restores the content of the tables (rows)
 
+  * :doc:`restore schema <restore-schema>` - restores the ScyllaDB cluster schema
 
-``vars.yaml`` file would look like:
+Each of those types has required prerequisites and follow-up actions.
+For more information, please read given restore type documentation.
 
-.. code::
+If both the schema and the content of the tables need to be restored, you must start with restoring the schema. Only after the schema is successfully restored can you proceed with restoring the content of the tables.
 
-   # backup_location specifies the location parameter used in Scylla Manager
-   # when scheduling a backup of a cluster.
-   backup_location: gcs:manager-bucket 
+Features
+========
```

Comment:
Question: can one choose a date to restore from, or is it always from the last version?
In either case, it needs to be explictly documented